packages/features/ee/organizations/components/OrganizationAvatar.tsx

```diff
@@ -0,0 +1,41 @@
+import classNames from "@calcom/lib/classNames";
+import { Avatar } from "@calcom/ui";
+import type { AvatarProps } from "@calcom/ui";
+
+type OrganizationAvatarProps = Pick<
+  AvatarProps,
+  "size" | "imageSrc" | "alt" | "indicator" | "gravatarFallbackMd5"
+> & {
+  organizationSlug?: string | null;
```

Comment:
I would prefer it to be required, so that we don't miss passing the `organizationSlug` property.
```suggestion
  organizationSlug: string | null | undefined;
```